django/db/models/query.py

```diff
@@ -1292,6 +1292,19 @@ def __init__(self, lookup, queryset=None, to_attr=None):
         self.queryset = queryset
         self.to_attr = to_attr
 
+    def __getstate__(self):
+        """
+        Modifies the pickling behaviour of the object.
+        """
+        obj_dict = self.__dict__.copy()
+        if self.queryset is not None:
+            # QuerySet should not be evaluated, see #27499
```

Comment:
We're more inclined to include ticket references in tests rather than in Django itself, and only for obscure issues where the behavior isn't straightforward. I think it could be skipped here. We always have `git blame` to look at history.